### ISSUE TITLE
Fatal error upon update to 2.8.4.1, might have occured earlier

### ISSUE THREAD

**Issue by mmages** [pods-framework/pods#6312]
### Description

Hello friends, long time no Pods! I have a site that is pretty creaky-old using Pods. Upon the update to the most recent version, I’m getting a fatal error when attempting to edit Pod data. Error text as follows:

WordPress version 5.8.2
Current plugin: Pods - Custom Content Types and Fields (version 2.8.4.1)
PHP version 7.4.25

Error Details
=============
An error of type E_ERROR was caused in line 388 of the file /path/to/site/wp-content/plugins/pods/classes/PodsField.php. Error message: Uncaught Error: Cannot use object of type Pod as array in /path/to/site/wp-content/plugins/pods/classes/PodsField.php:388
Stack trace:
#0 /path/to/site/wp-content/plugins/pods/classes/PodsField.php(329): PodsField->render_input_script(Object(stdClass))
#1 /path/to/site/wp-content/plugins/pods/classes/fields/pick.php(902): PodsField->input(‘pods_field_auth...’, Array, Array, Object(Pod), ‘40’)
#2 /path/to/site/wp-content/plugins/pods/classes/PodsForm.php(278): PodsField_Pick->input(‘pods_field_auth...’, Array, Array, Object(Pod), ‘40’)
#3 /path/to/site/wp-content/plugins/pods/ui/forms/table-row.php(37): PodsForm::field(‘pods_field_auth...’, Array, ‘pick’, Array, Object(Pod), ‘40’)
#4 /path/to/site/wp-content/plugins/pods/classes/PodsView.php(527): require(‘/www/michaelmcc...’)
#5 /path/to/site/wp-content/plugins/pods/classes/PodsView.php(118): PodsView::get_template

### Version

2.8.4.1

### Screenshots / Screencast

<img width="1183" alt="Screen Shot 2021-11-18 at 12 12 54 AM" src="https://user-images.githubusercontent.com/521749/142355998-fb52345c-7998-432e-b12f-13a35f2a4b97.png">

### Site Health Information

```text
`
### wp-core ###

version: 5.8.2
site_language: en_US
user_language: en_US
timezone: America/Denver
permalink: /%year%/%category%/%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: closed
environment_type: production
user_count: 4
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /www/michaelmccoydesign_326/public
wordpress_size: 47.65 MB (49967293 bytes)
uploads_path: /www/michaelmccoydesign_326/public/wp-content/uploads
uploads_size: 155.99 MB (163571386 bytes)
themes_path: /www/michaelmccoydesign_326/public/wp-content/themes
themes_size: 844.25 KB (864515 bytes)
plugins_path: /www/michaelmccoydesign_326/public/wp-content/plugins
plugins_size: 19.44 MB (20383360 bytes)
database_size: 9.53 MB (9994240 bytes)
total_size: 233.44 MB (244780794 bytes)

### wp-active-theme ###

name: McCoy Template (mccoy)
version: 1.2
author: Michael Arnold Mages
author_website: http://thinkmakedo.net/
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, editor-style, automatic-feed-links, menus, post-formats, custom-background, post-thumbnails, custom-header, widgets
theme_path: /www/michaelmccoydesign_326/public/wp-content/themes/mccoy
auto_update: Disabled

### wp-plugins-active (7) ###

Akismet Anti-Spam: version: 4.2.1, author: Automattic, Auto-updates enabled
Dropdown Menus: version: 1.0, author: Robert O'Rourke @ interconnect/it, Auto-updates enabled
Duplicator: version: 1.4.3, author: Snap Creek, Auto-updates enabled
FancyBox for WordPress: version: 3.3.3, author: Colorlib, Auto-updates enabled
Pods - Custom Content Types and Fields: version: 2.8.5, author: Pods Framework Team, Auto-updates enabled
Portfolio Image Pods UI: version: 0.1, author: Michael Arnold Mages, Auto-updates enabled
Really Simple SSL: version: 5.1.3, author: Really Simple Plugins, Auto-updates enabled

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
imagick_version: 3.5.1
file_uploads: File uploads is turned off
post_max_size: 128M
upload_max_filesize: 128M
max_effective_size: 128 MB
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 122 MB
	imagick::RESOURCETYPE_DISK: 1073741824
	imagick::RESOURCETYPE_FILE: 768
	imagick::RESOURCETYPE_MAP: 512 MB
	imagick::RESOURCETYPE_MEMORY: 256 MB
	imagick::RESOURCETYPE_THREAD: 1
imagemagick_file_formats: 3FR, 3G2, 3GP, AAI, AI, ART, ARW, AVI, AVS, BGR, BGRA, BGRO, BIE, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CR2, CRW, CUR, CUT, DATA, DCM, DCR, DCX, DDS, DFONT, DJVU, DNG, DOT, DPX, DXT1, DXT5, EPDF, EPI, EPS, EPS2, EPS3, EPSF, EPSI, EPT, EPT2, EPT3, ERF, EXR, FAX, FILE, FITS, FRACTAL, FTP, FTS, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, GV, H, HALD, HDR, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, IIQ, INFO, INLINE, IPL, ISOBRL, ISOBRL6, JBG, JBIG, JNG, JNX, JPE, JPEG, JPG, JPS, JSON, K25, KDC, LABEL, M2V, M4V, MAC, MAGICK, MAP, MASK, MAT, MATTE, MEF, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MRW, MSL, MSVG, MTV, MVG, NEF, NRW, NULL, ORF, OTB, OTF, PAL, PALM, PAM, PANGO, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PDF, PDFA, PEF, PES, PFA, PFB, PFM, PGM, PGX, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, PPM, PREVIEW, PS, PS2, PS3, PSB, PSD, PTIF, PWP, RADIAL-GRADIENT, RAF, RAS, RAW, RGB, RGBA, RGBO, RGF, RLA, RLE, RMF, RW2, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, SR2, SRF, STEGANO, SUN, SVG, SVGZ, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WEBP, WMF, WMV, WMZ, WPG, X, X3F, XBM, XC, XCF, XPM, XPS, XV, XWD, YCbCr, YCbCrA, YUV
gd_version: 2.3.0
gd_formats: GIF, JPEG, PNG, WebP, BMP, XPM
ghostscript_version: unknown

### wp-server ###

server_architecture: Linux 5.8.0-1038-gcp x86_64
httpd_software: nginx/1.21.3
php_version: 7.4.25 64bit
php_sapi: fpm-fcgi
max_input_variables: 10000
time_limit: 300
memory_limit: 256M
max_input_time: 300
upload_max_filesize: 128M
php_post_max_size: 128M
curl_version: 7.68.0 OpenSSL/1.1.1f
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.5.12-MariaDB-1:10.5.12+maria~focal-log
client_version: mysqlnd 7.4.25

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /www/michaelmccoydesign_326/public/wp-content
WP_PLUGIN_DIR: /www/michaelmccoydesign_326/public/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: nginx/1.21.3
pods-user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36
pods-session-save-path: /var/lib/php/sessions
pods-session-save-path-exists: No
pods-session-save-path-writable: No
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: Yes
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: Yes
pods-opcode-cache-redis: No
pods-object-cache-apc: Yes
pods-object-cache-apcu: Yes
pods-object-cache-memcache: No
pods-object-cache-memcached: No
pods-object-cache-redis: No
pods-memory-current-usage: 5.256M
pods-memory-current-usage-real: 4.000M
pods-network-wide: No
pods-install-location: /wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No
pods-sessions: Disable sessions
pods-can-use-sessions: No

`
```

### Pods Package

```json
{
    "meta": {
        "version": "2.8.5",
        "build": 1637212552
    },
    "pods": [
        {
            "name": "category",
            "id": 273,
            "label": "Category",
            "description": "",
            "show_in_menu": "0",
            "pod_index": "name",
            "_pods_1x_pod_id": "1",
            "type": "pod",
            "storage": "table",
            "groups": [
                {
                    "name": "more_fields",
                    "label": "More Fields",
                    "fields": [
                        {
                            "name": "name",
                            "id": 274,
                            "label": "Name",
                            "description": "",
                            "weight": 0,
                            "required": "1",
                            "text_max_length": "128",
                            "type": "text"
                        },
                        {
                            "name": "created",
                            "id": 275,
                            "label": "Date Created",
                            "description": "",
                            "weight": 1,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "modified",
                            "id": 276,
                            "label": "Date Modified",
                            "description": "",
                            "weight": 2,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "author",
                            "id": 277,
                            "label": "Author",
                            "description": "",
                            "weight": 3,
                            "pick_format_type": "single",
                            "pick_format_single": "autocomplete",
                            "default_value": "{@user.ID}",
                            "type": "pick",
                            "pick_object": "user"
                        },
                        {
                            "name": "slug",
                            "id": 278,
                            "label": "Permalink",
                            "description": "Leave blank to auto-generate",
                            "weight": 4,
                            "required": "0",
                            "_pods_1x_field_name": "slug",
                            "_pods_1x_field_id": "2",
                            "type": "slug"
                        },
                        {
                            "name": "outbound_link",
                            "id": 279,
                            "label": "Outbound link",
                            "description": "",
                            "weight": 5,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "outbound_link",
                            "_pods_1x_field_id": "7",
                            "text_max_length": "128",
                            "type": "text"
                        }
                    ]
                }
            ]
        },
        {
            "name": "portfolio_item",
            "id": 280,
            "label": "Portfolio Item",
            "description": "",
            "show_in_menu": "0",
            "pod_index": "name",
            "_pods_1x_pod_id": "2",
            "type": "pod",
            "storage": "table",
            "groups": [
                {
                    "name": "more_fields",
                    "label": "More Fields",
                    "fields": [
                        {
                            "name": "name",
                            "id": 281,
                            "label": "Name",
                            "description": "",
                            "weight": 0,
                            "required": "1",
                            "text_max_length": "128",
                            "type": "text"
                        },
                        {
                            "name": "created",
                            "id": 282,
                            "label": "Date Created",
                            "description": "",
                            "weight": 1,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "modified",
                            "id": 283,
                            "label": "Date Modified",
                            "description": "",
                            "weight": 2,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "author",
                            "id": 284,
                            "label": "Author",
                            "description": "",
                            "weight": 3,
                            "pick_format_type": "single",
                            "pick_format_single": "autocomplete",
                            "default_value": "{@user.ID}",
                            "type": "pick",
                            "pick_object": "user"
                        },
                        {
                            "name": "image",
                            "id": 285,
                            "label": "Image",
                            "description": "For best results, images should be cropped to a square before uploading",
                            "weight": 4,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "image",
                            "_pods_1x_field_id": "5",
                            "file_format_type": "multi",
                            "file_type": "any",
                            "type": "file"
                        },
                        {
                            "name": "order_by",
                            "id": 286,
                            "label": "Display Order",
                            "description": "Enter a number to control display order",
                            "weight": 5,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "order_by",
                            "_pods_1x_field_id": "16",
                            "number_decimals": "2",
                            "type": "number"
                        },
                        {
                            "name": "category",
                            "id": 287,
                            "label": "Category",
                            "description": "",
                            "weight": 6,
                            "required": "1",
                            "unique": "0",
                            "_pods_1x_field_name": "category",
                            "_pods_1x_field_id": "11",
                            "_pods_1x_sister_id": "0",
                            "pick_display": "{@name}",
                            "pick_size": "medium",
                            "pick_format_type": "single",
                            "pick_format_single": "dropdown",
                            "pick_limit": "1",
                            "type": "pick",
                            "pick_object": "pod",
                            "pick_val": "category"
                        },
                        {
                            "name": "slug",
                            "id": 288,
                            "label": "Permalink",
                            "description": "Leave blank to auto-generate",
                            "weight": 7,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "slug",
                            "_pods_1x_field_id": "4",
                            "type": "slug"
                        },
                        {
                            "name": "caption",
                            "id": 289,
                            "label": "Caption",
                            "description": "",
                            "weight": 8,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "caption",
                            "_pods_1x_field_id": "6",
                            "text_max_length": "128",
                            "type": "text"
                        },
                        {
                            "name": "co_designer",
                            "id": 290,
                            "label": "Co-designer",
                            "description": "",
                            "weight": 9,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "co_designer",
                            "_pods_1x_field_id": "8",
                            "text_max_length": "128",
                            "type": "text"
                        }
                    ]
                }
            ]
        },
        {
            "name": "publicity_category",
            "id": 302,
            "label": "Publicity Category",
            "description": "",
            "show_in_menu": "0",
            "pod_index": "name",
            "_pods_1x_pod_id": "4",
            "type": "pod",
            "storage": "table",
            "groups": [
                {
                    "name": "more_fields",
                    "label": "More Fields",
                    "fields": [
                        {
                            "name": "name",
                            "id": 303,
                            "label": "Name",
                            "description": "",
                            "weight": 0,
                            "required": "1",
                            "text_max_length": "128",
                            "type": "text"
                        },
                        {
                            "name": "created",
                            "id": 304,
                            "label": "Date Created",
                            "description": "",
                            "weight": 1,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "modified",
                            "id": 305,
                            "label": "Date Modified",
                            "description": "",
                            "weight": 2,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime"
                        },
                        {
                            "name": "author",
                            "id": 306,
                            "label": "Author",
                            "description": "",
                            "weight": 3,
                            "pick_format_type": "single",
                            "pick_format_single": "autocomplete",
                            "default_value": "{@user.ID}",
                            "type": "pick",
                            "pick_object": "user"
                        },
                        {
                            "name": "slug",
                            "id": 307,
                            "label": "Permalink",
                            "description": "Leave blank to auto-generate",
                            "weight": 4,
                            "required": "0",
                            "_pods_1x_field_name": "slug",
                            "_pods_1x_field_id": "18",
                            "type": "slug"
                        }
                    ]
                }
            ]
        },
        {
            "name": "publicity",
            "id": 291,
            "label": "Publicity Items",
            "description": "",
            "show_in_menu": "1",
            "pod_index": "name",
            "type": "pod",
            "storage": "table",
            "label_singular": "Publicity Item",
            "hierarchical": "0",
            "ui_style": "post_type",
            "menu_position": "0",
            "ui_actions_enabled": [
                "add",
                "edit",
                "duplicate",
                "delete"
            ],
            "ui_reorder_field": "menu_order",
            "ui_fields_manage": {
                "0": "name",
                "2": "modified"
            },
            "groups": [
                {
                    "name": "more_fields",
                    "label": "More Fields",
                    "fields": [
                        {
                            "name": "name",
                            "id": 292,
                            "label": "Name",
                            "description": "",
                            "weight": 0,
                            "required": "1",
                            "text_max_length": "128",
                            "type": "text",
                            "unique": "0",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "created",
                            "id": 293,
                            "label": "Date Created",
                            "description": "",
                            "weight": 1,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime",
                            "required": "0",
                            "unique": "0",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "datetime_allow_empty": "1",
                            "datetime_html5": "0",
                            "datetime_repeatable": "0",
                            "datetime_time_format_24": "hh_mm"
                        },
                        {
                            "name": "modified",
                            "id": 294,
                            "label": "Date Modified",
                            "description": "",
                            "weight": 2,
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A",
                            "type": "datetime",
                            "required": "0",
                            "unique": "0",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "datetime_allow_empty": "1",
                            "datetime_html5": "0",
                            "datetime_repeatable": "0",
                            "datetime_time_format_24": "hh_mm"
                        },
                        {
                            "name": "author",
                            "id": 295,
                            "label": "Author",
                            "description": "",
                            "weight": 3,
                            "pick_format_type": "single",
                            "pick_format_single": "autocomplete",
                            "default_value": "{@user.ID}",
                            "type": "pick",
                            "pick_object": "user",
                            "required": "0",
                            "unique": "0",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "pick_format_multi": "checkbox",
                            "pick_limit": "0",
                            "pick_taggable": "0",
                            "pick_allow_html": "0"
                        },
                        {
                            "name": "category",
                            "id": 296,
                            "label": "Category",
                            "description": "",
                            "weight": 4,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "category",
                            "_pods_1x_field_id": "19",
                            "_pods_1x_sister_id": "0",
                            "pick_display": "{@name}",
                            "pick_size": "medium",
                            "pick_format_type": "single",
                            "pick_format_single": "dropdown",
                            "pick_limit": "1",
                            "type": "pick",
                            "pick_object": "pod",
                            "pick_val": "publicity_category",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "pick_format_multi": "checkbox",
                            "pick_taggable": "0",
                            "pick_allow_html": "0"
                        },
                        {
                            "name": "file",
                            "id": 297,
                            "label": "File",
                            "description": "upload article text as PDF (optional)",
                            "weight": 5,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "file",
                            "_pods_1x_field_id": "15",
                            "file_format_type": "multi",
                            "file_type": "any",
                            "type": "file",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "file_uploader": "attachment",
                            "file_attachment_tab": "type",
                            "file_edit_title": "1",
                            "file_limit": "0",
                            "file_restrict_filesize": "10MB",
                            "file_linked": "0",
                            "file_add_button": "Add File",
                            "file_modal_title": "Attach a file",
                            "file_modal_add_button": "Add File"
                        },
                        {
                            "name": "slug",
                            "id": 298,
                            "label": "Permalink",
                            "description": "Leave blank to auto-generate",
                            "weight": 6,
                            "required": "0",
                            "unique": "0",
                            "input_helper": "input_helper_hide_slug",
                            "_pods_1x_field_name": "slug",
                            "_pods_1x_field_id": "10",
                            "type": "slug",
                            "admin_only": "0",
                            "restrict_capability": "0"
                        },
                        {
                            "name": "date",
                            "id": 299,
                            "label": "Date",
                            "description": "",
                            "weight": 7,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "date",
                            "_pods_1x_field_id": "12",
                            "type": "datetime",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "datetime_format": "mdy",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mma",
                            "datetime_allow_empty": "1",
                            "datetime_html5": "0",
                            "datetime_repeatable": "0",
                            "datetime_time_format_24": "hh_mm"
                        },
                        {
                            "name": "katherine",
                            "id": 300,
                            "label": "With Katherine McCoy",
                            "description": "",
                            "weight": 8,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "katherine",
                            "_pods_1x_field_id": "13",
                            "type": "boolean",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "boolean_format_type": "checkbox",
                            "boolean_yes_label": "Yes",
                            "boolean_no_label": "No"
                        },
                        {
                            "name": "peter",
                            "id": 407,
                            "label": "With Peter Stathis",
                            "description": "",
                            "weight": 9,
                            "type": "boolean",
                            "required": "0",
                            "unique": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "phone_enable_phone_extension": "1",
                            "paragraph_allow_html": "1",
                            "paragraph_oembed": "0",
                            "paragraph_wptexturize": "1",
                            "paragraph_convert_chars": "1",
                            "paragraph_wpautop": "1",
                            "paragraph_allow_shortcode": "0",
                            "wysiwyg_oembed": "0",
                            "wysiwyg_wptexturize": "1",
                            "wysiwyg_convert_chars": "1",
                            "wysiwyg_wpautop": "1",
                            "wysiwyg_allow_shortcode": "0",
                            "wysiwyg_max_length": "0",
                            "code_allow_shortcode": "0",
                            "boolean_format_type": "checkbox",
                            "boolean_yes_label": "Yes",
                            "boolean_no_label": "No",
                            "admin_only": "0",
                            "restrict_capability": "0"
                        },
                        {
                            "name": "dale",
                            "id": 301,
                            "label": "With Dale Fahnstrom",
                            "description": "",
                            "weight": 10,
                            "required": "0",
                            "unique": "0",
                            "_pods_1x_field_name": "dale",
                            "_pods_1x_field_id": "14",
                            "type": "boolean",
                            "admin_only": "0",
                            "restrict_capability": "0",
                            "boolean_format_type": "checkbox",
                            "boolean_yes_label": "Yes",
                            "boolean_no_label": "No"
                        },
                        {
                            "name": "curt",
                            "id": 456,
                            "label": "With Curt Fentress",
                            "description": "",
                            "weight": 11,
                            "type": "boolean",
                            "required": "0",
                            "unique": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "phone_enable_phone_extension": "1",
                            "paragraph_allow_html": "1",
                            "paragraph_oembed": "0",
                            "paragraph_wptexturize": "1",
                            "paragraph_convert_chars": "1",
                            "paragraph_wpautop": "1",
                            "paragraph_allow_shortcode": "0",
                            "wysiwyg_media_buttons": "1",
                            "wysiwyg_oembed": "0",
                            "wysiwyg_wptexturize": "1",
                            "wysiwyg_convert_chars": "1",
                            "wysiwyg_wpautop": "1",
                            "wysiwyg_allow_shortcode": "0",
                            "code_allow_shortcode": "0",
                            "pick_allow_html": "0",
                            "boolean_format_type": "checkbox",
                            "boolean_yes_label": "Yes",
                            "boolean_no_label": "No",
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ]
                        }
                    ]
                }
            ]
        }
    ],
    "templates": [
        {
            "name": "home_portfolio_list",
            "code": "<?php\nif ( $this->get_field('image') ) { ?>\n\t\t\t\t<div class=\"portfolio_item\">\n\t\t\t\t\t<a href=\"\/{@category.slug}\/\"><img src=\"{@image,image\/thumbnail}\" alt=\"{@name}\" height=\"300px\" width=\"300px\" class=\"portfolio_image\"\/><\/a>\n\t\t\t\t<\/div>\n\t\t\t\t<?php\n\t\t\t}\n?>",
            "id": 309,
            "type": "template",
            "_edit_lock": "1521644638:1"
        },
        {
            "name": "portfolio_list",
            "code": "<?php\nif ( $this->get_field('image') ) { ?>\n\t\t\t\t<div class=\"portfolio_item\">\n\t\t\t\t\t<a href=\"{@image.guid}\" class=\"lightbox\"><img src=\"{@image,image\/thumbnail}\" alt=\"{@name} {@co_designer}\" title=\"{@name} {@co_designer, add_with}\"  height=\"300px\" width=\"300px\" class=\"portfolio_image\"\/><\/a>\n\t\t\t\t\t<div class=\"caption\">{@caption}<?php\nif ( $this->get_field('co_designer') ) { ?>*<?php } ?><\/div>\n\t\t\t\t<\/div>\n\t\t\t\t<?php\n\t\t\t}\n?>",
            "id": 308,
            "type": "template"
        },
        {
            "name": "publicity_list",
            "code": "<div class=\"publicity_item\">\n\t<?php \n\t$file = $this->get_field('file');\n\tif ( $file ) { ?>\n\t\t<a href=\"{@file.guid}\">{@name}<\/a> <?php \/\/ print_r($file);\n\/\/ echo filesize( $file[0]['guid'] );?>\n\t<?php\n\t} else { ?>\n\t\t{@name}\n\t<?php\n\t}\n\t ?>\n<\/div>",
            "id": 310,
            "type": "template"
        }
    ],
    "pages": [
        {
            "name": "Arconas",
            "code": "",
            "id": 371,
            "_edit_last": "1",
            "_edit_lock": "1370536023:1",
            "page_template": "page-portfolio.php"
        },
        {
            "name": "Articles",
            "code": "<div id=\"primary\">\r\n\t<div id=\"content\" role=\"main\">\r\n\t\t<article class=\"page type-page status-publish hentry\">\r\n\t\t\t<header class=\"entry-header\">\r\n\t\t\t\t<h1 class=\"entry-title\">Articles<\/h1>\r\n\t\t\t<\/header><!-- .entry-header -->\r\n\r\n\t\t\t<div class=\"entry-content\">\r\n\r\n\t\t\t\t<?php\r\n\r\n\t\t\t\t$order = 't.date DESC';\r\n\t\t\t\t$where = 'category.slug = \"article\"';\r\n\r\n\r\n\t\t\t\t$Record = new Pod('publicity');\r\n\t\t\t\t$Record->findRecords($order, 99, $where);\r\n\t\t\t\tif($Record->getTotalRows() > 0) {\r\n\t\t\t\techo $Record->showTemplate('publicity_list');\r\n\t\t\t\t} else {\r\n\t\t\t\techo '<p>There are no publicity items.<\/p>';\r\n\t\t\t\t}\r\n\t\t\t\t\/\/ echo $Record->getPagination();\r\n\t\t\t\t?>\r\n\t\t\t<\/div>\r\n\t\t<\/article>\r\n\t<\/div>\r\n<\/div>",
            "id": 387,
            "_edit_last": "1",
            "_edit_lock": "1370410169:1",
            "page_title": "Articles"
        },
        {
            "name": "awards-collections",
            "code": "<div id=\"primary\">\r\n\t<div id=\"content\" role=\"main\">\r\n\t\t<article class=\"page type-page status-publish hentry\">\r\n\t\t\t<header class=\"entry-header\">\r\n\t\t\t\t<h1 class=\"entry-title\">Awards &amp; Collections<\/h1>\r\n\t\t\t<\/header><!-- .entry-header -->\r\n\t\t\t<h3>Awards<\/h3>\r\n\t\t\t<div class=\"entry-content\">\r\n\r\n\t\t\t\t<?php\r\n\r\n\t\t\t\t$order = 't.date DESC';\r\n\t\t\t\t$where = 'category.slug = \"award\"';\r\n\r\n\r\n\t\t\t\t$Record = new Pod('publicity');\r\n\t\t\t\t$Record->findRecords($order, 99, $where);\r\n\t\t\t\tif($Record->getTotalRows() > 0) {\r\n\t\t\t\techo $Record->showTemplate('publicity_list');\r\n\t\t\t\t} else {\r\n\t\t\t\techo '<p>There are no publicity items.<\/p>';\r\n\t\t\t\t}\r\n\t\t\t\t\/\/ echo $Record->getPagination();\r\n\t\t\t\t?>\r\n\t\t\t<\/div>\r\n\t\t\t<h3>Collections<\/h3>\r\n\t\t\t<div class=\"entry-content\">\r\n\r\n\t\t\t\t<?php\r\n\r\n\t\t\t\t$order = 't.date DESC';\r\n\t\t\t\t$where = 'category.slug = \"collections\"';\r\n\r\n\r\n\t\t\t\t$Record = new Pod('publicity');\r\n\t\t\t\t$Record->findRecords($order, 99, $where);\r\n\t\t\t\tif($Record->getTotalRows() > 0) {\r\n\t\t\t\techo $Record->showTemplate('publicity_list');\r\n\t\t\t\t} else {\r\n\t\t\t\techo '<p>There are no collections.<\/p>';\r\n\t\t\t\t}\r\n\t\t\t\t\/\/ echo $Record->getPagination();\r\n\t\t\t\t?>\r\n\t\t\t<\/div>\r\n\t\t<\/article>\r\n\t<\/div>\r\n<\/div>",
            "id": 389,
            "_edit_last": "1",
            "_edit_lock": "1370583735:1",
            "page_title": "Awards & Collections"
        },
        {
            "name": "Corner Office (Pablo\/Haworth)",
            "code": "",
            "id": 505,
            "_edit_lock": "1520957495:1",
            "_edit_last": "1",
            "page_title": "Corner Office (Pablo\/Haworth)",
            "page_template": "page-portfolio.php",
            "pod": "0"
        },
        {
            "name": "furniture",
            "code": "",
            "id": 311,
            "page_template": "page-portfolio.php",
            "type": "page",
            "_edit_lock": "1369951764:1"
        },
        {
            "name": "humanscale",
            "code": "",
            "id": 313,
            "page_template": "page-portfolio.php",
            "type": "page",
            "_edit_lock": "1520882541:1"
        },
        {
            "name": "interiors",
            "code": "",
            "id": 314,
            "page_template": "page-portfolio.php",
            "type": "page"
        },
        {
            "name": "knoll",
            "code": "",
            "id": 312,
            "page_template": "page-portfolio.php",
            "type": "page"
        },
        {
            "name": "mccoy-collection",
            "code": "",
            "id": 316,
            "page_template": "page-portfolio.php",
            "type": "page"
        },
        {
            "name": "products-interiors",
            "code": "",
            "id": 315,
            "page_template": "page-portfolio.php",
            "type": "page",
            "_edit_lock": "1369953354:1",
            "_edit_last": "1"
        }
    ]
}
```

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
I've checked the debug backtrace, it should work fine since a Pod object is array accessible right?

**Comment by sc0ttkclark** [comment]
I'll check this out today and see what could possibly be happening here. ArrayAccess should just work for a Whatsit Pod object.

**Comment by sc0ttkclark** [comment]
I believe this is now fixed in daf3c5ecfc55f9a8a21e93a6091fb0b9769b1593

This fix will go out in Pods 2.8.6 but for now, consider adjusting your code to avoid using Pod() objects and follow the 1.x to 2.x guide here: https://docs.pods.io/compare-functionality-in-pods-1-x-to-pods-2-x/
